asab/storage/elasticsearch.py

```diff
@@ -83,6 +114,42 @@ def session(self):
 		return self._ClientSession
 
 
+	async def is_connected(self) -> bool:
+		"""Check if the service is connected to ElasticSearch cluster.
+
+		Raises:
+			ConnectionError: Connection failed.
+
+		Returns:
+			bool: True if the service is connected.
+		"""
+		for url in self.ServerUrls:
+			try:
+				async with self.session().request(
+					method="GET",
+					url=url,
+					ssl=self.SSLcontext,
+					headers=self.Headers,
+				) as resp:
+					await self.session().close()
+					if resp.status not in {200, 201}:
+						resp = await resp.json()
+						L.error("Failed to connect to ElasticSearch.", struct_data={
+							"code": resp.get("status"),
+							"reason": resp.get("error", {}).get("reason")
+						})
+						return False
+
```

Comment:
Won't it just return False after encountering a faulty node? It won't go further (won't check the rest of the urls).